.gitignore

```diff
@@ -6,3 +6,4 @@ pipelines/.gradle
 pipelines/gradle-cache
 pipelines/target
 **/.DS_Store
+build-farm/platform-specific-configurations/platformConfigFile.sh
```

Comment:
Why is this file being ignored?

Reply:
It's an automatically generated file by `set-platform-config.sh` and shouldn't be commited during builds